src/Microsoft.Security.Utilities.Core/PreciselyClassifiedSecurityKeys/SEC101_102_AdoPat.cs

```diff
@@ -5,14 +5,14 @@
 
 namespace Microsoft.Security.Utilities
 {
-    public class AdoPat : RegexPattern
+    public class AdoLegacyPat : RegexPattern
```

Comment:
>AdoLegacyPat [](http://example.com/codeflow?start=17&length=12)

This rule no longer applies to the latest ADO PAT format and thefore I'm renaming it.